src/services/services.ts

```diff
@@ -3345,6 +3345,12 @@ export function createLanguageService(
         );
     }
 
+    function getImports(fileName: string): readonly string[] {
+        synchronizeHostData();

```

Comment:
If you are using synchronize host data this is not syntax operation. Need to add this to `invalidOperationsInPartialSemanticMode`

Need to add protocol command to `invalidPartialSemanticModeCommands`

Reply:
I don't know if I need to synchronize host data to call getResolvedModuleFromModuleSpecifier. How can I tell?
(I'm guessing the answer is yes, though.)

Reply:
If you need to use more than open file , it needs to synchronize data .. that is anything that uses program needs to synchronize host data